common/shared.c

```diff
@@ -251,3 +251,90 @@ void show_perf_mr(int tsize, int iters, struct timespec *start,
 	printf("usec/xfer: %f", ((float)elapsed / iters / xfers_per_iter));
 	printf(" }\n");
 }
+
+void ft_csusage(char *name, char *desc)
+{
+	fprintf(stderr, "Usage:\n");
+	fprintf(stderr, "  %s [OPTIONS]\t\tstart ping server\n", name);
+	fprintf(stderr, "  %s [OPTIONS] <host>\tpong given host\n", name);
+
+	fprintf(stderr, "\nOptions:\n");
+	fprintf(stderr, "  -n\tdomain name\n");
+	fprintf(stderr, "  -p\tport number\n");
+	fprintf(stderr, "  -f\tspecific provider\n");
```

Comment:
Will it be better to add some examples of the provider names? Also, it's not clear from the description - "specific provider" what the user needs to specify. 

-f <specific provider name> (e.g. psm)